### PR TITLE
Fix Iot Edge Quickstart E2E tests

### PR DESCRIPTION
**Pull request by philipktlin**
1. fix temp filter docker file to ensure that pid 1 is used for running temp filter.
2. increase timeout for result verification.
3. add log messages.
4. remove username and password when print out configure command for debugging.

### CODE REVIEW

File: smoke/IotEdgeQuickstart/details/IotedgedWindows.cs [hunk 1]
```diff
@@ -219,28 +221,28 @@ public Task Start()
 
                     if (iotedgeService.Status != ServiceControllerStatus.Running)
                     {
-                        throw new Exception("Can't start up iotedge service within timeout period.");
+                        throw new Exception("Can't start iotedge service within timeout period.");
                     }
+
+                    // Add delay to ensure iotedge service is completely started.
+                    await Task.Delay(new TimeSpan(0, 0, 0, 5));
```
Comment (varunpuranik) [inline smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: Nit: TimeSpan.FromSeconds for readability..

Comment (varunpuranik) [inline smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: Why is this needed? Because iotedged is running but hasn't started the servers yet?

File: smoke/IotEdgeQuickstart/details/IotedgedWindows.cs [hunk 2]
```diff
@@ -249,35 +251,38 @@ public async Task Stop()
...
+                        iotedgeService.WaitForStatus(ServiceControllerStatus.Stopped, TimeSpan.FromMinutes(2));
+                        iotedgeService.Refresh();
+
+                        if (iotedgeService.Status != ServiceControllerStatus.Stopped)
+                        {
+                            throw new Exception("Can't stop iotedge service within timeout period.");
+                        }
+
+                        // Add delay to ensure iotedge service is completely stopped.
+                        await Task.Delay(new TimeSpan(0, 0, 0, 5));
```
Comment (varunpuranik) [inline smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: Wait, iotedged is stopped but not completely stopped? What does that mean?

File: smoke/IotEdgeQuickstart/details/IotedgedWindows.cs (hunk 1 above)
Comment (philipktlin) [reply smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: iotedged is running; but it is not ready to accept operation request; such as iotedge list.  Therefore 5 sec delay is added.

File: smoke/IotEdgeQuickstart/details/IotedgedWindows.cs (hunk 2 above)
Comment (philipktlin) [reply smoke/IotEdgeQuickstart/details/IotedgedWindows.cs]: I should remove it since I changed the stop method to use WaitForStatus instead.
